src/utils/stringUtils.c

```diff
@@ -257,7 +261,12 @@ int strSubStringCase(const char* h, const char* n) {
   if (h == NULL || n == NULL) {
     return 0;
   }
+  #ifdef __MINGW32__
```

Comment:
Here I could not come up with an equivalent of strcasestr for Windows. Any ideas?

Reply:
Convert both strings to all lowercase and then compare with `strstr`:

This code should work (untested):
```c
char* strlower(const char* str) {
  char* lower = oidc_strcopy(str);
  for(int i = 0; lower[i]; i++){
    lower[i] = tolower(lower[i]);
  }
  return lower;
}

// strSubStringCase part
char* h_l = strlower(h);
char* n_l = strlower(h);
int ret = strstr(h,n);
secFree(h_l);
secFree(n_l);
return ret;
```

Reply:
I'll add this to the branch I created